store/mangoStore.ts

```diff
@@ -1124,6 +1137,59 @@ const mangoStore = create<MangoStore>()(
             state.client = newClient
           })
         },
+        estimatePriorityFee: async (feeMultiplier) => {
+          const set = get().set
+          const group = mangoStore.getState().group
+          const client = mangoStore.getState().client
+
+          await get().actions.reloadMangoAccount()
```

Comment:
This is already being updated via websocket on every change. I think since reloading the account is a pretty heavy call and you're calling estimatePriorityFee every 30 secs it might be best to remove this. 

Will the rest of the function work without this call?

Reply:
reloading the mangoAccount is not necessary for priority fees -- it has been removed